src/main/kotlin/no/nav/syfo/client/pdl/PipPersondataResponse.kt

```diff
@@ -0,0 +1,113 @@
+package no.nav.syfo.client.pdl
+
+data class PipPersondataResponse(
+    val person: PipPerson,
+    val identer: PipIdenter,
+    val geografiskTilknytning: PipGeografiskTilknytning?,
+)
+
+data class PipPerson(
+    val adressebeskyttelse: List<PipAdressebeskyttelse>,
+    val doedsfall: List<PipDoedsfall>,
+)
+
+data class PipAdressebeskyttelse(
+    val gradering: Gradering,
+)
+
+enum class Gradering {
+    STRENGT_FORTROLIG_UTLAND,
+    STRENGT_FORTROLIG,
+    FORTROLIG,
+    UGRADERT,
+}
+
+fun PipPersondataResponse.isKode6(): Boolean {
```

Comment:
Kunne alle disse vært member functions i stedet for extention functions?

Reply:
Tror vi har vært innom dette før: Argumentene mot er 1. dette er en data-klasse, og 2. Dette er en dto som "hører" til en annen app. Mottakelig for motargumenter (men synes egentlig ikke forskjellen er så stor).

Reply:
Sant det, aller mest enig i at dette blir riktig basert på argument nr 2 👍🏼

Reply:
Virker fornuftig å bruke extension-functions når det er en dto vi ikke "eier" selv.